src/Nri/Ui/Highlighter/V1.elm

```diff
@@ -302,50 +302,74 @@ update msg model =
                 ( model, Cmd.none )
 
 
+nextInteractive : Int -> List (Highlightable marker) -> Int
+nextInteractive index highlightables =
+    if Maybe.map .type_ (List.Extra.getAt index highlightables) == Just Highlightable.Interactive then
```

Comment:
I wonder if we could change `nextInteractive` and `previousInteractive` to take the _current_ index, instead of the index offset by 1 -- I'm thinking that we if did this (and changed the implementation to use a fold rather than recursion), then the places we use `nextInteractive` and `previousInteractive` could be simpler.

I think `foldl`/`foldr` would be more performant here -- at least, I think there would be fewer operations in most cases: `List.Extra.getAt` will need to do an operation at least `index` number of times  ([`getAt`](https://github.com/elm-community/list-extra/blob/8.6.0/src/List/Extra.elm#L125) uses [`drop`](https://github.com/elm/core/blob/1.0.5/src/List.elm#L622), which recurses through the undesired elements of the linked list to remove them.